includes/admin/reporting/widgets/class.llms.analytics.widget.sold.php

```diff
@@ -112,14 +107,14 @@ public function set_query() {
 						{$txn_meta_join}
 						JOIN {$wpdb->postmeta} AS sales ON sales.post_id = txns.ID
 						WHERE
-						        ( txns.post_status = 'llms-txn-succeeded' )
+						        ( txns.post_status = 'llms-txn-succeeded' OR txns.post_status = 'llms-txn-refunded' )
```

Comment:
This will incorrectly include the full amount of the original transaction in the Net Sales amount, rather than the difference between the original amount and refunded.

Reply:
We've changed the SELECT portion to get the sum of the amount minus any refunded amounts.